frontend/src/pages/Policy/policyContent/FaqsContent.tsx

```diff
@@ -71,111 +74,53 @@ export const policyFaqs = [
-            </li>
-          </ul>
-        </p>
-
-        <p>
-          <strong>Gun Deaths (Young Adults, 18-25):</strong>
-          <ul>
-            <li>
-              <strong>Breakdowns by race/ethnicity:</strong> Yes
-            </li>
-            <li>
-              <strong>Breakdowns by age:</strong> No
-            </li>
-            <li>
-              <strong>Breakdowns by sex:</strong> No
-            </li>
-            <li>
-              <strong>Breakdowns by urbanicity:</strong> No
-            </li>
-          </ul>
-        </p>
-
-        <p>
-          <strong>Gun Homicides (Black Men-specific):</strong>
-          <ul>
-            <li>
-              <strong>Breakdowns by race/ethnicity:</strong> No
-            </li>
-            <li>
-              <strong>Breakdowns by age:</strong> Yes
-            </li>
-            <li>
-              <strong>Breakdowns by sex:</strong> No
-            </li>
-            <li>
-              <strong>Breakdowns by urbanicity:</strong> Yes
-            </li>
-          </ul>
-        </p>
-
         <p>
-          <strong>Gun Homicides:</strong>
-          <ul>
-            <li>
-              <strong>Breakdowns by race/ethnicity:</strong> Yes
-            </li>
-            <li>
-              <strong>Breakdowns by age:</strong> Yes
-            </li>
-            <li>
-              <strong>Breakdowns by sex:</strong> Yes
-            </li>
-            <li>
-              <strong>Breakdowns by urbanicity:</strong> No
-            </li>
-          </ul>
+          Currently, all of our gun violence datasets include national- and
+          state-level data.
         </p>
-
+        <DatasetList datasets={datasets} />
         <p>
-          <strong>Gun Suicides:</strong>
-          <ul>
-            <li>
-              <strong>Breakdowns by race/ethnicity:</strong> Yes
-            </li>
-            <li>
-              <strong>Breakdowns by age:</strong> Yes
-            </li>
-            <li>
-              <strong>Breakdowns by sex:</strong> Yes
-            </li>
-            <li>
-              <strong>Breakdowns by urbanicity:</strong> No
-            </li>
-          </ul>
+          Additionally, (if the data is available) users can filter the
+          datasets' demographics by:
         </p>
+        <ul className='py-0 my-0'>
+          <li className='py-0 my-0'>
+            <strong>Age Groups:</strong> Available in 5-year increments and
+            single-year groups (ranging from 0 to 85+ years, including unknown
+            ages).
+          </li>
+          <li>
+            <strong>Sex:</strong> Data can be filtered for male and female
+            demographics.
+          </li>
+          <li>
+            <strong>Race and Ethnicity:</strong> Options include White, Black,
+            American Indian/Alaska Native, Asian, Hawaiian Native/Pacific
+            Islander, more than one race, unknown race, and Hispanic.
```

Comment:
I'm not sure how much detail we should do here since it's sort of duplicating the methodology, but should we indicate that these race groups are non-Hispanic ?